packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -584,6 +615,24 @@ const filterCreator: RemoteFilterCreator = ({ client, config }) => ({
       .filter(setting => setting.invalidManagedBySaltoField !== undefined)
       .map(createInvalidManagedBySaltoFieldFetchWarning)
 
+    const typesOfFetchedInstances = new Set(
+      elements
+        .filter(isInstanceElement)
+        .map(instance => instance.getTypeSync())
+    )
```

Comment:
```suggestion
    const typesOfFetchedDataInstances = new Set(
      elements
        .filter(isInstanceElement)
        .map(instance => instance.getTypeSync())
        .filter(objectType => isCustomObjectSync(objectType))
    )
```
Isn't this logic relevant for Data instances only?

Reply:
I dunno, is it?

Reply:
It is :)

Reply:
Ack